ipn/ipnlocal/expiry.go

```diff
@@ -119,3 +120,94 @@ func (em *expiryManager) flagExpiredPeers(netmap *netmap.NetworkMap) {
 		peer.Key = key.NodePublicWithBadOldPrefix(peer.Key)
 	}
 }
+
+// nextPeerExpiry returns the time that the next node in the netmap expires
```

Comment:
May want to document that the zero value for `time.Time` is a possible return value.

Reply:
Good idea; added